tools/BUILD.bazel

```diff
@@ -0,0 +1,15 @@
+load("@rules_jvm_external//:defs.bzl", "artifact")
```

Comment:
looks like we are not using `artifact`!!

Reply:
Good find--removed.